autopush/endpoint.py

```diff
@@ -246,7 +246,7 @@ def _router_fail_err(self, fail):
         fail.trap(RouterException)
         exc = fail.value
         if exc.log_exception:
-            log.err(fail, **self._client_info)
+            log.err(fail, **self._client_info)  # pragma nocover
```

Comment:
Tests should mimic reality, and via this bug, we know this line works. Setting `nocover` for now.